config.php

```diff
@@ -1,16 +1,15 @@
 <?php
 
-
 /** @noinspection MissedFieldInspection */
 
 use humhub\modules\fcmPush\Events;
 use humhub\components\Controller;
-
-//use humhub\modules\notification\widgets\NotificationInfoWidget;
 use humhub\modules\user\widgets\AuthChoice;
+use humhub\widgets\BaseStack;
```

Comment:
Required?

Reply:
Removed in https://github.com/humhub/fcm-push/pull/44/commits/56b77e7a5046def63b8c7a6f46cab11b4ceacdd4